packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1525,6 +1525,24 @@ mixin WidgetInspectorService {
     return false;
   }
 
+  /// Notify debuggers to open an inspector on the object.
+  void _sendInspectEvent(Object? object){
+    inspect(object);
+
+    final _Location? location = WidgetInspectorService.instance._getSelectedSummaryWidgetLocation(null, 'my-group');
+    if (location != null){
+      postEvent(
+        'navigate',
+        <String, Object>{
+          'file': location.file,
+          'line': location.line,
+          'column': location.column,
```

Comment:
It would be good to add some comments here that document the intended formats here. There's often been confusion in the past with URIs vs paths and zero-based vs one-based line/columns, so having the _expected_ format/values written here would help understand where the bug is they don't seem as expected:

```
          'file': location.file, // File path (Win: "C:\foo\bar.dart", Linux/Mac: "/foo/bar.dart")     _or_     File URI
           // 1-based     _or_     0-based
          'line': location.line,
          'column': location.column,
```

Alternatively, changing `file` to `filePath` or `fileUri` would work for that too.